bin/process_bulk_metadata.py

```diff
@@ -0,0 +1,335 @@
+                opened_at = issue.created_at.strftime("%Y-%m-%d")
+                if verbose:
+                    print(
+                        f"ISSUE {issue.number} ({opened_at}): {issue.title} {issue.html_url}",
+                        file=sys.stderr,
+                    )
+
+                # Parse metadata changes from issue
+                json_block = self._parse_metadata_changes(issue.body)
+                if not json_block:
+                    if close_old_issues:
+                        # for old issues, filed without a JSON block, we append a comment
+                        # alerting them to how to file a new issue using the new format.
+                        # If possible, we first parse the Anthology ID out of the title:
+                        # Metadata correction for {anthology_id}. We can then use this to
+                        # post a link to the original paper so they can go through the
+                        # automated process.
+                        anthology_id = None
+                        match = re.search(r"Paper Metadata: [\{]?(.*)[\}]?", issue.title)
+                        if match:
+                            anthology_id = match[1]
+                        if anthology_id:
+                            print(
+                                f"-> Closing issue {issue.number} with a link to the new process",
+                                file=sys.stderr,
+                            )
+                            url = f"https://aclanthology.org/{anthology_id}"
+                            issue.create_comment(
+                                f"### Notice\n\nThe Anthology has had difficulty keeping up with the large number of metadata corrections we receive. We have therefore updated our workflow with a more automatated process. We are closing this issue, and ask that you help us out by recreating your request using this new workflow. You can do this by visiting [the paper page associated with this issue]({url}) and clicking on the yellow 'Fix metadata' button. This will take you through a few simple steps."
+                            )
+                            # close the issue as "not planned"
+                            issue.edit(state="closed", state_reason="not_planned")
+                            continue
+                    else:
+                        if verbose:
+                            print("-> Skipping (no JSON block)", file=sys.stderr)
+                    continue
+
+                # Skip issues that are not approved by team member
+                if not skip_validation and not self._is_approved(issue):
+                    if verbose:
+                        print("-> Skipping (not approved yet)", file=sys.stderr)
+                    continue
+
+                anthology_id = json_block.get("anthology_id")
+                collection_id, _, _ = deconstruct_anthology_id(anthology_id)
+                xml_path = f"data/xml/{collection_id}.xml"
+
```

Comment:
I usually do something similar, but I assumed that this was unnecessary since the file is being open by `self.repo.get_contents`, and would therefore be relative to the repo root. But I tried it from a different directory and it fails, so I'll add this.

Reply:
Looks like I had to create both: one path relative to the repo root (for reading from a branch with the GitHub module), and another absolute path (for writing to the actual XML file). I tested this by running in both `bin/` and root.